### ISSUE TITLE
Ublock origin needs to be reloaded when opening new tabs from a site on chrome.

### ISSUE THREAD

**Issue by madmax2** [uBlockOrigin/uBlock-issues#611]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description
While browsing a site such as youtube.
When I am opening some new tabs by clicking a link on the current tab using one of these methods
-right click a link to open new tab
-ctrl click a link to open it in new tab

I get message about ublock 
"Reload page to use extension"
Essentially ublock is turned off or broken when a new tab is opened from an existing tab of a site via a link

Please how to fix this problem?

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]
https://www.youtube.com

### Steps to Reproduce

1. Using chrome, Go to a site e.g youtube
2. Ensure that the ublock origin is active/enabled for the site
3. Right click or ctrl click to open a new tab from any of the videos showing on youtube

### Expected behavior:

[What you expected to happen]
Ublock origin should show that it is enabled/active and blocking the ads

### Actual behavior:

[What actually happened]
Ublock origin actually showing that is disabled or broken (with the white circle icon) on the extension bar  on chrome and ads are showing on the new tab from the same site

### Your environment

* uBlock Origin version: 1.19.6
* Browser Name and version: chrome 74.0.3729.169
* Operating System and version:
windows 10 pro v1903

**Comment by gwarser** [comment]
> (with the white circle icon) 

Like this: 

![image](https://user-images.githubusercontent.com/886325/58706642-f20f0a00-83b2-11e9-836a-ff11d593c3aa.png)


Make sure it's enabled on all sites:

![image](https://user-images.githubusercontent.com/886325/58706605-db68b300-83b2-11e9-82f8-d75ad6ea096e.png)

**Comment by madmax2** [comment]
@gwarser I uninstall and reinstall the extension and it seems to working as normal again.

Thanks...
Your pic is also helpful cos I think I might have change the site access to On Click and didn't know how it would affect the extension. I thought it would still work as normal.
I also forgot I changed that setting,

I just tested it again now and change site access to "On Click"
and can confirm the exact problem returns.
When I changed it to "On all sites", the extension works as normal again.

So this was the cause of the problem - changing site access to "On click"
